spec/features/merchant_items/index_spec.rb

```diff
@@ -7,6 +7,7 @@
 
         @merchant_2 = Merchant.create(name: "McDonalds")
         @item_2 = @merchant_2.items.create(name: "Big Mac", description: "Juicy", unit_price: 29, merchant_id: @merchant_2.id)
+        require 'pry'; binding.pry
```

Comment:
Should this pry be removed before pushing to main?

Reply:
Yep! Good catch, thanks